tests/release/e2e-test-default-bundle.go

```diff
@@ -0,0 +1,249 @@
+package release
+
+import (
+	"strings"
+	"time"
+
+	"github.com/google/uuid"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/redhat-appstudio/e2e-tests/pkg/constants"
+	"github.com/redhat-appstudio/e2e-tests/pkg/framework"
+	"github.com/redhat-appstudio/e2e-tests/pkg/utils"
+	"github.com/redhat-appstudio/e2e-tests/pkg/utils/tekton"
+
+	ecp "github.com/hacbs-contract/enterprise-contract-controller/api/v1alpha1"
+	appstudiov1alpha1 "github.com/redhat-appstudio/release-service/api/v1alpha1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	klog "k8s.io/klog/v2"
+	"knative.dev/pkg/apis"
+)
+
+var managednamespaceSecret = []corev1.ObjectReference{
+	{Name: redhatAppstudioUserSecret},
+}
+
+var roleRules = map[string][]string{
+	"apiGroupsList": {""},
+	"roleResources": {"secrets"},
+	"roleVerbs":     {"get", "list", "watch"},
+}
+
+var paramsReleaseStrategyDefault = []appstudiov1alpha1.Params{
+	{Name: "extraConfigGitUrl", Value: "https://github.com/scoheb/strategy-configs.git"},
+	{Name: "extraConfigPath", Value: "m6.yaml"},
+	{Name: "extraConfigRevision", Value: "main"},
+}
+
+var _ = framework.ReleaseSuiteDescribe("[HACBS-738]test-release-service-happy-path", Label("release", "defaultBundle"), func() {
+	defer GinkgoRecover()
+	// Initialize the tests controllers
+	framework, err := framework.NewFramework()
+	Expect(err).NotTo(HaveOccurred())
+
+	var defaultEcPolicy = ecp.EnterpriseContractPolicySpec{}
+	var devNamespace = uuid.New().String()
+	var managedNamespace = uuid.New().String()
+
+	var cm = &corev1.ConfigMap{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: buildPipelineBundleDefaultName,
+		},
+		Data: map[string]string{
+			"default_build_bundle": buildPipelineBundleDefault,
+		},
+	}
+
+	BeforeAll(func() {
+		kubeController := tekton.KubeController{
+			Commonctrl: *framework.CommonController,
+			Tektonctrl: *framework.TektonController,
+		}
+
+		demo, err := framework.CommonController.CreateTestNamespace(devNamespace)
```

Comment:
`CreateTestNamespace` returns a namespace. It would make more sense to me in the returned value.name was used to reference the namespace in the other spots in the code instead of relying on the predefined string

Reply:
I'm not sure I got the comment